akka-samples/akka-docs-java-lambda/pom.xml

```diff
@@ -31,6 +31,11 @@
       <version>1.0-SNAPSHOT</version>
     </dependency>
     <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-stream-testkit-experimental_2.10</artifactId>
```

Comment:
interesting, what will happen if this is run with 2.11?
doesn't have to be solved now